test/test_integration.py

```diff
@@ -1184,6 +1185,54 @@ def on_click(n_clicks):
         time.sleep(1)
         self.snapshot("Tabs 2 rendered ")
 
+        # do some extra tests while we're here
+        # and have access to Graph and plotly.js
+        self.check_graph_config_shape()
+        self.check_plotlyjs()
+
+    def check_plotlyjs(self):
+        # find plotly.js files in the dist folder, check that there's only one
+        all_dist = os.listdir(dcc.__path__[0])
+        js_re = r'^plotly-(.*)\.min\.js$'
+        plotlyjs_dist = [fn for fn in all_dist if re.match(js_re, fn)]
+
+        self.assertEqual(len(plotlyjs_dist), 1)
```

Comment:
I don't know how we ended up with 2 copies of plotly.js on master, 1.44.1 and 1.45.0... the associated PRs all seem to swap one file for another, maybe a mis-merge somewhere. But anyway this test will prevent that in the future.